packages/experiments/src/components/TilesList/TilesList.tsx

```diff
@@ -185,12 +186,15 @@ export class TilesList<TItem> extends React.Component<ITilesListProps<TItem>, IT
       // For each cell at the start of a grid.
       const grid = cells[i].grid;
 
-      const isPlaceholder = grid.isPlaceholder;
+      const { isPlaceholder, maxRowCount } = grid;
```

Comment:
maxRowCount is limit how many rows TilesList can have or how many cells max per Row can have?

Reply:
It is meant to limit the number of rows the TileList can have.

Reply:
The number of cells in a row is determined by the cell size and the available width. So `maxRowCount` controls the number of rows displayed within a 'grid', which is a set of contiguous cells. There is currently not a limit to the number of 'grids' allocated to the list.